#!/usr/bin/env python3

PACKAGE = "pablowsky_perception"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("min_h", int_t, 0, "min hue value", 50, 0,  255)
gen.add("max_h", int_t, 0, "max hue value", 150, 0, 255)

gen.add("min_s", int_t, 0, "min saturation value", 50, 0, 255)
gen.add("max_s", int_t, 0, "max saturation value", 150, 0, 255)

gen.add("min_v", int_t, 0, "min v value", 50, 0, 255)
gen.add("max_v", int_t, 0, "max v value", 150 , 0,  255)

exit(gen.generate(PACKAGE, "pablowsky_perception", "hsvBounds"))
